mlir/lib/Bindings/Python/IRCore.cpp

```diff
@@ -6,6 +6,7 @@
 //
 //===----------------------------------------------------------------------===//
 
+#include <filesystem>
```

Comment:
I believe this broke the bot, see here: https://lab.llvm.org/buildbot/#/builders/116/builds/9857